go/examples/createuser/main.go

```diff
@@ -26,18 +27,29 @@ func main() {
 
 	c := utils.NewClient()
 
-	payload := UserCreatePayload{
+	perBucket := UserCreatePayload{
 		Username: os.Args[2],
 		Password: os.Args[3],
-		Access: []UserCreateAccess{
+		Buckets: []UserCreateAccess{
 			{
 				Name:  os.Args[4],
-				Roles: []string{"data_writer", "data_reader"},
+				Roles: []string{"data_writer"},
 			},
 		},
 	}
 
-	resp, err := c.Do(http.MethodPost, "/v2/clusters/"+os.Args[1]+"/users", payload)
+	resp, err := c.Do(http.MethodPost, "/v2/clusters/"+os.Args[1]+"/users", perBucket)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	_ = utils.PrettyPrint(resp)
+
+	allBucket := UserCreatePayload{
+		AllBucketsAccess: "data_reader",
+	}
+
+	resp, err = c.Do(http.MethodPost, "/v2/clusters/"+os.Args[1]+"/users/"+os.Args[2], allBucket)
```

Comment:
We're changing it to `put`
```suggestion
	resp, err = c.Do(http.MethodPut, "/v2/clusters/"+os.Args[1]+"/users/"+os.Args[2], allBucket)
```

Reply:
@Rhymond Updated it to Put. But, in the docs it is still a Post request.